plugins/src/main/kotlin/extension/DependencyHandleScope.kt

```diff
@@ -103,8 +103,12 @@ fun DependencyHandlerScope.allLibrariesImpl() {
 }
 
 fun DependencyHandlerScope.allServicesImpl() {
+    // For analytics configuration, either use noop, or use the impl, with at least one analyticsproviders implementation
+    // implementation(project(":services:analytics:noop"))
```

Comment:
We could try using some config value like:

```kotlin
if (project.property("disable-analytics") == "true") {
    implementation(project(":services:analytics:noop"))
} else {
    implementation(project(":services:analytics:impl"))
}
```

Or some other custom value we could add to a config module maybe? We don't really need it now though, it's just a suggestion for the future.

Reply:
Yes, good point, I would love to have a common place where all the possible tweaks, configurations and settings are centralized... At least the project is quite well architectured to be able to do whatever we want in the future regarding that.